lndr-backend/data/lndr-server.config.test

```diff
@@ -68,6 +68,7 @@ aws {
     photo-bucket = "lndr-avatars"
     access-key-id = "AAAAAAAAAAAAAAAAAAAA"
     secret-access-key= "6444444444444444444444444444444444444444"
+    notification-api-key= "bobloblawlobslawbomb"
```

Comment:
🎨Missing a space here before the `=`. Same goes for L70, which it looks like we overlooked in an earlier PR.